x-pack/plugins/security/public/authentication/components/authentication_state_page/authentication_state_page.tsx

```diff
@@ -8,11 +8,12 @@ import { EuiIcon, EuiSpacer, EuiTitle } from '@elastic/eui';
 import React from 'react';
 
 interface Props {
+  className?: string;
```

Comment:
__note:__ motivation behind that change is that I wanted to make body of this component wider for access notice as we have much more text there (`600px` instead of `480px`)